ocaml/xapi/xapi_globs.ml

```diff
@@ -633,6 +633,10 @@ let network_reset_trigger = "/tmp/network-reset"
 
 let first_boot_dir = "/etc/firstboot.d/"
 
+(** {2 Xenopsd metadata persistence} *)
```

Comment:
True, this would be a nice change. Probably best done independently of a feature PR though.